xrootd/protocol/protocol/protocol.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Package protocol contains the structures describing request and response
+for protocol request (see XRootD specification).
+
+Response consist of 3 parts:
+
+1) GeneralResponse - general response that is always returned and specifies protocol version and flags describing server type.
+
+2) SecurityInfo - a response part that is added to general response
+if `ReturnSecurityRequirements` is provided and server supports it.
+It contains security version, security options, security level,
+and number of following security overrides, if any.
+
+3) A list of SecurityOverride - alterations needed to the specified predefined security level.
+*/
+package protocol // import "go-hep.org/x/hep/xrootd/protocol/protocol"
+
+// RequestID is the id of the request, it is sent as part of message. See xrootd protocol specification for details.
```

Comment:
> See xrootd protocol specification for details.

please add either a link and/or the name of the section that deals with these details.